### PR TITLE
Improve output feedback

### PR DESCRIPTION
**Pull request by ghost**
Add `force` flag. Will sort files even if they're already sorted
image

Add index in front of output
image

Colorize the output
red (dryRun): file is not sorted
image

yellow (!dryRun): file was not sorted, sorting applied
green: file was already sorted
image

blue (force): file was sorted, but is force sorted
image

Add `indent` option: amount of spaces to use for indentation when sorting
Add `unsorted-only` option: Will only print files that are not sorted (dryRun) or that were sorted.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,74 +1,121 @@
...
 
                 // CI check mode. Compare sorted file and local file.
-                if (JSON.stringify(jsonContent) !== JSON.stringify(sortedJsonContent)) {
-                    throw new Error(`${file} not sorted alphabetically`);
+                if (stringContent !== sortedStringContent) {
+                    console.error(`[${index}] ${fileName} not sorted`.red);
+                    unsortedFiles.push(file);
+                } else if (!unsortedOnly) {
+                    console.info(`[${index}] ${fileName} is sorted`.green);
                 }
```
Comment (tiemenrtuinstra) [inline index.js]: It is not clear if the output is from the --dry-run so I would suggest to change the wording and maybe the colors.

File: index.js [hunk 2]
```diff
@@ -1,74 +1,121 @@
...
+            if (unsortedFiles.length > 0) {
+                if (dryRun) {
+                    console.info(`\n${unsortedFiles.length} of ${files} file(s) not sorted`.red);
+                     process.exit(1);
+                }
+                console.info(`\n${unsortedFiles.length} files sorted`.yellow.bold);
+                process.exit(0);
+            }
+
+            console.info(`\nAll files (${files.length}) are sorted`.green.bold);
```
Comment (tiemenrtuinstra) [inline index.js:76]: Maybe a bit of over engineering but I would suggest a different text for normal, forced, or dry-run

**Review by tiemenrtuinstra — COMMENTED** [review]
Looks good but some small comments for thought,

File: index.js (hunk 2 above)
Comment (ghost) [reply index.js:76]: Yes, feels a bit too much. Makes code less readable). Had this before 😁

**Review by youssef-almardini — APPROVED** [review]

**Pull request merged**